src/utils/dotnetUtils.ts

```diff
@@ -80,6 +81,17 @@ export namespace dotnetUtils {
         const regExp: RegExp = new RegExp(`<${prop}>(.*)<\\/${prop}>`);
         const matches: RegExpMatchArray | null = (await projFile.getContents()).match(regExp);
         if (!matches) {
+            // if we can't find the targetFramework in the csproj, there may be a `Directory.Build.props` file that sets it
+            const buildPropsFile: string = 'Directory.Build.props';
+            const buildPropsFiles = await workspace.findFiles(buildPropsFile, null);
+            if (buildPropsFiles.length > 0) {
+                const buildPropsContents = (await AzExtFsExtra.readFile(buildPropsFiles[0])).toString();
```

Comment:
Is the `.toString()` necessary here / doesn't `readFile` already return a string?

Reply:
You are correct. I have a feeling copilot added that lol